newdle/client/src/components/creation/FinalStep.js

```diff
@@ -4,6 +4,7 @@ import {useHistory} from 'react-router';
 import {t, Trans} from '@lingui/macro';
 import PropTypes from 'prop-types';
 import {Button, Container, Header, Icon, Input, Checkbox} from 'semantic-ui-react';
+import {useIsMobile} from 'src/util/hooks';
```

Comment:
I think we use relative imports everywhere else... add `"javascript.preferences.importModuleSpecifier": "relative",` to your vscode settings.